_pages/fsa/install/building-the-fsa-storefront-from-libraries-4-0.md

```diff
@@ -0,0 +1,127 @@
+
+***
+
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+***
+
+## Prerequisites
+
+Before carrying out the procedures below, ensure the following front-end and back-end requirements are in place.
+
+### Front-End Development Requirements
+
+Your Angular development environment should include the following:
+
+- **Angular CLI**: Version 12.0.5 or later.
+- **node.js**: Version 14.x is recommended.
+- **yarn**: Version 1.22.x or later.
+
+### Back-End Server Requirements
+
+FSA Spartacus uses SAP Commerce and Financial Services Accelerator back end and makes use of the sample data.
+
+- **SAP Commerce version**: Release **2105** (the latest patch is recommended).
+- **Financial Services Accelerator version**: Release **2202** is required.
+
+For more information, see [Installing SAP Commerce Cloud FSA for use with FSA Spartacus]({{ site.baseurl }}{% link _pages/fsa/install/installing-sap-commerce-with-fsa-spartacus.md %}).
+
+### Windows Setup
+
+To successfully install the project in the Windows environment, first make sure that you have installed GitBash.
+
+Next, create the .npmrc file at the root of your Angular project.
+The .npmrc file should contain the following:
+
+```shell
+shell = "{instalation directory}\\Git\\bin\\bash.exe" 
+script-shell = "{instalation directory}\\Git\\bin\\bash.exe" 
+
+
+Example:
+shell = "C:\\Program Files\\Git\\bin\\bash.exe"
+script-shell = "C:\\Program Files\\Git\\bin\\bash.exe"
+```
+
+Once you have configured this, you should execute all the commands in the procedures that follow from the GitBash console.
+
+## Creating a New Angular App
+
+The following procedure describes how to create a new Angular application with the name `mystore`.
+
+1. Open a terminal or command prompt window at the location of your choice.
+2. Using the Angular CLI, generate a new Angular application with the following command:
+
+   ```bash
+   ng new mystore --style=scss
+   ```
+
+   When prompted for Angular routing, enter `n` for 'no'.
+
+   The `mystore` folder and the new app are created.
+
+3. Access the newly created `mystore` folder with the following command:
+
+     ```bash
+     cd mystore
+     ```
+
+## FSA Spartacus Project Setup
+
+The easiest way to start a new project is to use Angular Schematics to set up your application quickly.
+
+You can add FSA Spartacus libraries to your Spartacus Angular project by running the following command from your project root:
+
+```shell
+ng add @spartacus/fsa-schematics --baseSite=sample-financial-site --currency=usd,eur --language=en,de,fr
+```
+
+For a detailed explanation, see [FSA Schematics]({{ site.baseurl }}{% link _pages/fsa/install/fsa-schematics.md %}).
+
+For a full list of available parameters, see Spartacus schematics [documentation](https://github.com/SAP/spartacus/tree/develop/projects/schematics).
+
+**NOTE**: After you finish this step, you need to upgrade your FSA Spartacus application to version 4.0. 
```

Comment:
Is an upgrade needed during the initial setup of the project? This part is needed when the customer is already on FSA SPA 3.0.0 for example and then they need to upgrade it to the latest. During the first setup, it's enough just to specify FSA versions in package.json (alongside with Spartacus ones) and that would be enough, right?

Reply:
@SrckyD Maybe you can explain better the problem we have here?